Add section about stray files (#1)
sigurdm/site-www: src/content/guides/workspaces.md

must have a SDK version constraint of `^3.6.0` or higher.
:::

<a name='stray-files'></a>
## Stray files

> When you are migrating an existing mono-repo to use Pub workspaces, there will

Reviewer: ```suggestion
When you are migrating an existing monorepo to use pub workspaces, there will
```

be existing "stray" `pubspec.lock` and `.dart_tool/package_config.json` files
adjacent to each pubspec. These would shadow the `pubspec.lock` and

> `.dart_tool/package_config.json` placed next to the root

Reviewer: ```suggestion
`.dart_tool/package_config.json` placed next to the root.
```

> Therefore `pub get` will delete any `pubspec.lock` and

Reviewer: ```suggestion
Therefore, `pub get` will delete any `pubspec.lock` and
```

> `.dart_tool/package_config.json` located in directories between the root and
> (including) any workspace package.

Reviewer: ```suggestion
`.dart_tool/package_config.json` located in directories between the root,
including any workspace package.
```

Does that keep the same meaning? I think the "and" might be redudant but you can make the call.

```
/
  pubspec.yaml                       # Root
  packages/
    pubspec.lock                     # Deleted by `pub get`
    .dart_tool/package_config.json   # Deleted by `pub get`
    foo/
      pubspec.yaml                   # Workspace member
      pubspec.lock                   # Deleted by `pub get`
      .dart_tool/package_config.json # Deleted by `pub get`
```

If any directory between the workspace root and a workspace package contains a

> "stray" `pubspec.yaml` file that is not member of the workspace, `pub get` will

Reviewer: ```suggestion
"stray" `pubspec.yaml` file that is not a member of the workspace, `pub get` will
```
